WooCommerce/Classes/ViewRelated/Products/Product Loader/ProductVariationLoadUseCase.swift

```diff
@@ -0,0 +1,72 @@
+import Yosemite
+
+/// Encapsulates the async logic for loading product variation and related data (its parent product) for UI display.
+final class ProductVariationLoadUseCase {
```

Comment:
Do you think that it worth moving this class under Yosemite? It seems related, also if here we don't do storage operations.

Reply:
In Yosemite, we can't easily dispatch actions to different stores via `StoresManager` though - in this use case, it dispatches actions to both `ProductStore` and `ProductVariationStore`. If the actions are all in the same store, it certainly makes sense!